libs/top_level_metric_risk_levels.py

```diff
@@ -18,8 +18,12 @@ def calc_risk_level(value: Optional[float], thresholds: List[float]) -> RiskLeve
     Returns:
         A RiskLevel.
     """
-    assert len(thresholds) == 3, "Must pass low, med and high thresholds."
-    level_low, level_med, level_high = thresholds
+    assert len(thresholds) in [3, 4], "Must pass low, med and high thresholds."
+    if len(thresholds) == 3:
+        level_low, level_med, level_high = thresholds
+        level_extreme = math.inf
+    else:
+        level_low, level_med, level_high, level_extreme = thresholds
```

Comment:
Why is the last threshold not level_critical?

Reply:
that risk level is actually titled "critical" in the enum, so would be a misleading name